src/components/previewCard/imageHelper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+
+interface Props {
+  imageSrc: string
+  pictureAlt: string
+  isLogo: boolean
+  handleOnErrorImage?: (e) => void
+}
+
+const Image: FC<Props> = ({
+  imageSrc,
+  pictureAlt,
+  handleOnErrorImage,
+  isLogo,
+}) => (
+  <img
+    src={imageSrc}
+    alt={pictureAlt}
+    className={
+      isLogo
+        ? "flex-shrink-0 mr-10"
+        : `${styles.promotionImage} w-12/12 object-cover`
+    }
+    onError={(e) => {
+      handleOnErrorImage && handleOnErrorImage(e)
+    }}
+  />
```

Comment:
I would totally duplicate that `img` tag and not extract it. now we just need a condition `isLogo`. if it was not extracted, the styles would just be inlined where they're used

Reply:
Yes I dropped the "imageHelper", it is more straightforward this way.